src/topology/algebra/ring.lean

```diff
@@ -106,6 +102,10 @@ variables [topological_ring α]
   makes the product into a topological ring. -/
 instance {β : Type*}
   [ring β] [topological_space β] [topological_ring β] : topological_ring (α × β) :=
+{ }
+
+instance {β : Type*} {C : β → Type*} [Π b, topological_space (C b)]
+  [Π b, ring (C b)] [∀ b, topological_ring (C b)] : topological_ring (Π b, C b) :=
 { continuous_neg := continuous_neg }
```

Comment:
This instance duplicates the one just below it

Reply:
Yes, I messed up while merging master. Fixed.